app/models/setting/provisioning.rb

```diff
@@ -20,9 +20,20 @@ def self.load_defaults
         self.set('libvirt_default_console_address', N_("The IP address that should be used for the console listen address when provisioning new virtual machines via Libvirt"), "0.0.0.0"),
         self.set('update_ip_from_built_request', N_("Foreman will update the host IP with the IP that made the built request"), false),
         self.set('use_shortname_for_vms', N_("Foreman will use the short hostname instead of the FQDN for creating new virtual machines"), false),
+        self.set('websockets_encrypt', N_("VNC/SPICE websocket proxy console access encryption (websockets_ssl_key/cert setting required)"), SETTINGS[:require_ssl]),
```

Comment:
auth.rb in the same directory:

<pre>
14:00 $ grep websockets_ app/models/setting/*
app/models/setting/auth.rb:        self.set('websockets_encrypt', N_("Should Foreman encrypt websockets (VNC console access). Choose on, off or auto."), "auto"),
app/models/setting/auth.rb:        self.set('websockets_ssl_key', N_("Private key that Foreman will use to encrypt websockets "), nil),
app/models/setting/auth.rb:        self.set('websockets_ssl_cert', N_("Certificate that Foreman will use to encrypt websockets "), nil),
app/models/setting/provisioning.rb:        self.set('websockets_encrypt', N_("VNC/SPICE websocket proxy console access encryption (websockets_ssl_key/cert setting required)"), SETTINGS[:require_ssl]),
app/models/setting/provisioning.rb:        self.set('websockets_ssl_key', N_("Private key that Foreman will use to encrypt websockets "), nil),
app/models/setting/provisioning.rb:        self.set('websockets_ssl_cert', N_("Certificate that Foreman will use to encrypt websockets "), nil)
</pre>

Probably a bad rebase over fb293de.